rules/library.bzl

```diff
@@ -156,7 +156,7 @@ def _write_umbrella_header(
 
 """
 
-    for header in public_headers:
+    for header in public_headers + private_headers:
```

Comment:
I'm not sure about this... should the umbrella header include the private headers?

Reply:
Without this the error is 

```
While building module 'FW' imported from /private/var/tmp/_bazel_ob/c86c2b68df7abbb07c0dd5103a56923b/sandbox/darwin-sandbox/43/execroot/build_bazel_rules_ios/tests/ios/app/App/main.m:1:
In file included from <module-includes>:1:
/private/var/tmp/_bazel_ob/c86c2b68df7abbb07c0dd5103a56923b/sandbox/darwin-sandbox/43/execroot/build_bazel_rules_ios/bazel-out/ios-x86_64-min10.0-applebin_ios-ios_x86_64-dbg-ST-a7ed3b5187e2/bin/tests/ios/app/FW-modulemap/FW-umbrella.h:20:1: error: umbrella header for module 'FW' does not include header 'FW_Private.h' [-Werror,-Wincomplete-umbrella]
/private/var/tmp/_bazel_ob/c86c2b68df7abbb07c0dd5103a56923b/sandbox/darwin-sandbox/43/execroot/build_bazel_rules_ios/tests/ios/app/App/main.m:1:9: fatal error: could not build module 'FW'
@import FW;
 ~~~~~~~^~
2 errors generated.
```

Reply:
Looking a bit deeper I think this diff is wrong, the problem is that the private headers are being included in the `Headers` section of the VFS overlay...

Reply:
@ob yeah that problem is a manifestation of plugging in private headers to the interface where they shouldn't be! This PR seems to pass CI now BTW - maybe just fixing docs?

Reply:
I didn't have time to look at this today but I want to figure out why the VFS overlay is wrong.

Reply:
Added the real fix... now only the docs are failing.